cmd/armada/export/export.go

```diff
@@ -0,0 +1,18 @@
+package export
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/armada/cmd/armada/export/logs"
+	kind "sigs.k8s.io/kind/pkg/cluster"
+)
+
+// ExportCmd returns a new cobra.Command under root command for armada
+func ExportCmd(provider *kind.Provider) *cobra.Command {
```

Comment:
I'm not sure about the usefulness of this command. It does not add much to what kind does except executing it for every cluster you pass or have configs for.

I think a much more interesting command that could replace this would be an `execute` or `kind` command that passes the command to kind and executes it for every cluster. You could even import the commands from kind as subcommands to make it more interesting.